sdl/sdl.go

```diff
@@ -95,5 +139,22 @@ func ManifestVersion(manifest manifest.Manifest) ([]byte, error) {
 	}
 
 	sum := sha256.Sum256(sortedBytes)
-	return []byte(sum[:]), nil
+
+	return sum[:], nil
+}
+
+func (s *sdl) DeploymentGroups() ([]*dtypes.GroupSpec, error) {
+	if s.data == nil {
+		return []*dtypes.GroupSpec{}, errUninitializedConfig
+	}
+
+	return s.data.DeploymentGroups()
+}
+
+func (s *sdl) Manifest() (manifest.Manifest, error) {
+	if s.data == nil {
```

Comment:
might want to do `len(s.data) == 0` in these so there's no issue of the various kinds of `nil`